extensions/wikia/Wall/css/Wall.scss

```diff
@@ -25,6 +25,12 @@ html>body #Wall .new-reply .speech-bubble-avatar { *position: absolute; *left:-9
 }
 
 .Wall {
+
+	/* overrides /resources/jquery/jquery.makeCollapsible.css */
+	li .mw-collapsible-toggle {
+		float: right;
```

Comment:
I didn't find anything in the app repo that would set it to other value than right or none, so I guess it is correct